src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -32,6 +32,8 @@ public abstract class Element : IDisposable, IGraphicItem, IFormattable
         /// <param name="init"></param>
         protected void SafeInit(Action init)
         {
+            TransactionManager.Instance.EnsureInTransaction(DocumentManager.Instance.CurrentDBDocument);
```

Comment:
@ikeough 
I am more curious about why there is not an active transaction even when we are going to create an element. Does this mean when a task is running, no transactions have been opened? Or does this mean some one has closed the transaction without reopening it?

But as EnsureIntransaction is just ensuring that there is an active transaction, it will make the Revit nodes work. I will agree with the changes with the questions in mind.